src/crawlee/sessions/_cookies.py

```diff
@@ -0,0 +1,193 @@
+from __future__ import annotations
+
+from copy import deepcopy
+from http.cookiejar import Cookie
+from typing import Any, ClassVar, Literal, cast
+
+from httpx import Cookies
+from typing_extensions import NotRequired, Required, TypedDict, override
+
+
+class BaseCookieParam(TypedDict, total=False):
+    name: Required[str]
+    value: Required[str]
+    domain: NotRequired[str]
+    path: NotRequired[str]
+    secure: NotRequired[bool]
+
+
+class CookieParam(BaseCookieParam, total=False):
+    http_only: NotRequired[bool]
+    expires: NotRequired[int]
+    same_site: NotRequired[Literal['Lax', 'None', 'Strict']]
+
+
+class PWCookieParam(BaseCookieParam, total=False):
```

Comment:
We haven't used the "PW" anywhere so far. Could you please use the full name - `PlaywrightCookieParam`, instead?